src/themes/classic/components/input.js

```diff
@@ -14,52 +24,29 @@ const appearances = {
       boxShadow: theme =>
         `inset 0 0 0 1px ${theme.palette.red.base}, inset 0 1px 2px ${theme.scales.neutral.N4A}`
     },
-    _placeholder: {
-      color: 'scales.neutral.N6A'
-    },
     _focus: {
-      outline: 'none',
       boxShadow: theme =>
         `inset 0 0 2px ${theme.scales.neutral.N4A}, inset 0 0 0 1px ${theme.scales.blue.B7}, ${theme.shadows.focusRing}`
     },
     _disabled: {
-      cursor: 'not-allowed',
-      boxShadow: theme => `inset 0 0 0 1px ${theme.scales.neutral.N4A}`,
-      backgroundColor: 'scales.neutral.N2'
+      boxShadow: theme => `inset 0 0 0 1px ${theme.scales.neutral.N4A}`
     }
   },
   neutral: {
     backgroundColor: 'scales.neutral.N2A',
     _invalid: {
       boxShadow: theme => `inset 0 0 0 1px ${theme.palette.red.base}`
     },
-    _placeholder: {
-      color: 'scales.neutral.N6A'
-    },
     _focus: {
-      outline: 'none',
       backgroundColor: 'white',
       boxShadow: theme => `0 0 0 2px ${theme.scales.blue.B6A}`
     },
     _disabled: {
-      cursor: 'not-allowed',
-      boxShadow: theme => `inset 0 0 0 1px ${theme.scales.neutral.N4A}`,
-      backgroundColor: 'scales.neutral.N2'
+      boxShadow: theme => `inset 0 0 0 1px ${theme.scales.neutral.N4A}`
     }
   },
   none: {
-    backgroundColor: 'white',
-    _invalid: {},
-    _placeholder: {
-      color: 'scales.neutral.N6A'
-    },
-    _focus: {
-      outline: 'none'
```

Comment:
Was this just a no-op style?

Reply:
`outline: none` is now included in the internal styles (maybe it should be moved to theme `baseStyles` tho